docs/introduction/usage.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: API
+title: Usage
 firstpage:
 ---
 
-## API
+## Usage
 
 ```{include} ../../README.md
 :start-after: <!-- start snippet-usage -->
```

Comment:
This is duplicated with the API pages no?

Reply:
It is. On PZ they do it like this, where they have installation in the **Usage** tab, as well as basic API instructions. Should I merge installation with this, or remove it?

Reply:
I would do a page for install and point to pages for API for basic usage.